src/NBitcoin/ValidatedAddress.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NBitcoin
+{
+    public class ValidatedAddress
```

Comment:
I'm not sure whether we should be adding to NBitcoin like this @DanGould . We are in the process of porting over the library to the full node ;)

Reply:
I think this is OK, just add `[JsonProperty(PropertyName = "isvalid")]` on top of the property in order to define the exact format of this json field.

Reply:
👍